src/languages/css-visitor-keys.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Visitor keys for the CSS Tree AST.
+ * @author Nicholas C. Zakas
+ */
+
+export const visitorKeys = {
+	AnPlusB: [],
+	Atrule: ["prelude", "block"],
+	AtrulePrelude: ["children"],
+	AttributeSelector: ["name", "value"],
+	Block: ["children"],
+	Brackets: ["children"],
+	CDC: [],
+	CDO: [],
+	ClassSelector: [],
+	Combinator: [],
+	Comment: [],
+	Condition: ["children"],
+	Declaration: ["value"],
+	DeclarationList: ["children"],
+	Dimension: [],
+	Feature: ["value"],
+	FeatureFunction: ["value"],
+	FeatureRange: ["left", "middle", "right"],
+	Function: ["children"],
+	GeneralEnclosed: ["children"],
+	Hash: [],
+	IdSelector: [],
+	Identifier: [],
+	Layer: [],
+	LayerList: ["children"],
+	MediaQuery: ["condition"],
+	MediaQueryList: ["children"],
```

Comment:
```suggestion
	MediaQueryList: ["children"],
	NestingSelector: [],
```

https://github.com/csstree/csstree/blob/master/docs/ast.md#nestingselector